Userland/Utilities/sed.cpp

```diff
@@ -261,14 +261,25 @@ struct SArguments {
         if (delimiter == '\n' || delimiter == '\\')
             return SedError::parsing_error(lexer, "\\n and \\ cannot be used as delimiters."sv);
 
-        auto pattern = lexer.consume_until(delimiter);
+        auto pattern = lexer.consume_until([is_escape_sequence = false, &delimiter](char c) mutable {
```

Comment:
Not sure why you're capturing `delimiter` by reference here?

Reply:
What would be the better way? vscode did this on "quick fix" :D

Reply:
You could just have `delimiter` instead of `&delimiter`? Not sure what vscode would be fixing there.